src/Proto3/Suite/JSONPB/Class.hs

```diff
@@ -167,16 +181,27 @@ parseField :: FromJSONPB a
            => A.Object -> Text -> A.Parser a
 parseField = A.explicitParseField parseJSONPB
 
+instance HasDefault E.Encoding where
+  def       = E.pairs mempty
+  isDefault = E.nullEncoding
+
+instance HasDefault A.Value where
+  def       = A.Null
+  isDefault = (== A.Null)
+
 -- * JSONPB rendering and parsing options
 
 data Options = Options
   { optEmitDefaultValuedFields :: Bool
+  , optEmitInlinedOneof :: Bool
+  -- ^ for compatibility with Go JSONPB.
```

Comment:
There should be a much longer comment somewhere (possibly here, but could be somewhere else) explaining what is the issue with the Go JSONPB encoding.  Specifically, it should clarify that the JSONPB way of representing `oneOf` is incompatible with the OpenAPI v3.0 way of representing `oneOf`